sharktank/sharktank/models/llama/llama.py

```diff
@@ -137,6 +137,7 @@ def __init__(self, theta: Theta, config: LlamaModelConfig):
             "attention_embedding",
             RotaryEmbeddingLayer(
                 rope_dimension_count=hp.rope_dimension_count,
+                rope_freq_base=hp.rope_freq_base,
```

Comment:
Remove all instances of `hp.rope_freq_base`, if it's not added as an optional param.